files/install-deps-worker.yaml

```diff
@@ -14,6 +14,9 @@
       - python3-kubernetes # for sandcastle
       - python3-fedora # to access FAS
       - python3-requests
+
+      - python3-flask  # sorry jirko
+      - python3-flask-restplus
```

Comment:
You can add it to `setup.cfg` as well since it's a common (to both, server & worker) requirement now.

Reply:
> Are you f.... kidding me?
> 
> (you don't need to add `python3-flask` since it's pulled in with `python3-flask-restplus`)

but that's installed only in the service image, not worker, so yeah, I funkin' had to put it here in order to make this PR work

I agree, setup.cfg sounds like a better solution

Reply:
Ok, I think we are on a different track ...

> but that's installed only in the service image, not worker, so yeah, I funkin' had to put it here in order to make this PR work

I was suggesting to remove only `python3-flask` not `python3-flask-restplus`.

> I agree, setup.cfg sounds like a better solution

Important in my comment was "as well". Please leave `python3-flask-restplus` in `files/install-deps-worker.yaml` so we install RPM, not python package (even there should be no difference)